data/json/bionics.json

```diff
@@ -43,6 +43,8 @@
     "name": "Alloy Plating - Arms",
     "description": "The flesh on your arms has been surgically replaced by alloy plating.  Provides passive protection and can be used in conjunction with bionic martial arts.",
     "occupied_bodyparts": [ [ "ARM_L", 4 ], [ "ARM_R", 4 ] ],
+    "bash_protec": [ [ "ARM_L", 3 ], [ "ARM_R", 3 ] ],
```

Comment:
`bash_protec`? `bash_protect`?

Reply:
That's for consistency with existing `env_protec`.

Reply:
`bash_protec` as in `env_protec` and `cut_protec`